### PR TITLE
Prepare for release 0.26

### PR DESCRIPTION
**Pull request by rhuss**
Hold until 0.26 is out and we can fill in the hashsums.

### CODE REVIEW

File: source-kafka@0.25.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class SourceKafkaAT025 < Formula
+  v = "v0.25.0"
```
Comment (christophd) [inline source-kafka@0.25.rb:4]: Should this be v0.26.0?

File: admin@0.25.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class AdminAT025 < Formula
+  v = "v0.25.0"
```
Comment (christophd) [inline admin@0.25.rb:4]: Should this be v0.26.0?

**Review by christophd — COMMENTED** [review]
I have two questions about used versions

File: source-kafka@0.25.rb (hunk 1 above)
Comment (rhuss) [reply source-kafka@0.25.rb:4]: No, because that's the formula for version 0.25. 0.26 is in `source-kafka.rb`

File: admin@0.25.rb (hunk 2 above)
Comment (rhuss) [reply admin@0.25.rb:4]: dito.

**Review by maximilien — APPROVED** [review]
/approve
/lgtm

**Comment by rhuss** [comment]
Has been already done.

**Pull request closed without merging**